tests/src/test/scala/ch/epfl/bluebrain/nexus/tests/kg/IdResolutionSpec.scala

```diff
@@ -110,6 +115,20 @@ class IdResolutionSpec extends BaseSpec {
 
     "redirect to fusion resource selection page if text/html accept header is present (multiple result)" in { pending }
 
+    "redirect to delta resolve if the request comes to the proxy endpoint" in {
+      deltaClient.get[String](s"/resolve-proxy-pass/$neurosciencegraphSegment", Bob) { (_, response) =>
+        response.status shouldEqual StatusCodes.SeeOther
+        locationHeaderOf(response) shouldEqual deltaResolveEndpoint(encodedNeurosciencegraphId)
+      }(PredefinedFromEntityUnmarshallers.stringUnmarshaller)
+    }
+
+    "redirect to fusion resolve if the request comes to the proxy endpoint with text/html accept header is present" in {
+      deltaClient.get[String](s"/resolve-proxy-pass/$neurosciencegraphSegment", Bob, acceptTextHtml) { (_, response) =>
+        response.status shouldEqual StatusCodes.SeeOther
+        locationHeaderOf(response) shouldEqual fusionResolveEndpoint(encodedNeurosciencegraphId)
+      }(PredefinedFromEntityUnmarshallers.stringUnmarshaller)
+    }
```

Comment:
Dream assertion would be

`response shouldBe redirectTo(fusionResolveEndpoint(encodedNeurosciencegraphId))`

It would be really nice to get rid of this boilerplate too but I don't know how: `(PredefinedFromEntityUnmarshallers.stringUnmarshaller)`

Reply:
Yeah but with this option we'd be matching on the response body which is not really relevant